spec/cypress/integration/attendanceView.spec.js

```diff
@@ -0,0 +1,107 @@
+import faker from 'faker'
+import { createSelector } from '../utils'
+import dayjs from 'dayjs'
+import weekday from 'dayjs/plugin/weekday'
+
+dayjs.extend(weekday)
+
+const { name, internet } = faker
+const firstName = name.firstName()
+const fullName = name.findName(firstName)
+const email = internet.email(firstName)
+const password = internet.password()
+const checkIn = new Date(Date.now() - 21600000)
+const checkOut = new Date(Date.now() - 7200000)
+let childFullName
+
+const weekPickerText = (date = dayjs()) =>
+  `${date.weekday(0).format('MMM D')} - ${date.weekday(6).format('MMM D')}`
+
+describe('AttendanceView', () => {
+  beforeEach(() => {
+    cy.app('clean')
+    cy.appFactories([
+      [
+        'create',
+        'confirmed_user',
+        {
+          email,
+          full_name: fullName,
+          greeting_name: firstName,
+          password,
+          password_confirmation: password
+        }
+      ]
+    ]).then(users => {
+      cy.appFactories([['create', 'business', { user_id: users[0].id }]]).then(
+        businesses => {
+          cy.appFactories([
+            ['create', 'necc_child', { business_id: businesses[0].id }]
+          ]).then(children => {
+            childFullName = children[0].full_name
+            cy.appFactories([
+              ['create', 'child_approval', { child_id: children[0].id }]
+            ]).then(child_approvals => {
+              cy.appFactories([
+                [
+                  'create',
+                  'nebraska_hourly_attendance',
+                  {
+                    child_approval_id: child_approvals[0].id,
+                    check_in: checkIn,
+                    check_out: checkOut
+                  }
+                ]
+              ])
+            })
+          })
+        }
+      )
+    })
+    cy.intercept({
+      method: 'POST',
+      url: '/login'
+    }).as('login')
+
+    cy.visit('/login')
+    cy.get(createSelector('email')).type(email)
+    cy.get(createSelector('password')).type(password)
+    cy.get(createSelector('loginBtn')).click()
+
+    cy.intercept({
+      method: 'GET',
+      url: '/attendances'
+    }).as('attendances')
+    cy.visit('/attendance')
+  })
+
+  describe('content', () => {
+    it('renders content', () => {
+      cy.contains(childFullName)
+      cy.contains('4 hrs 0 mins')
+      cy.contains('no info')
```

Comment:
Can you indicate how many times you expect to see 'no info'?  My guess is there will be one day with an attendance and six days with no info, can you do (pseudocode): `cy.contains('no info').exactly(6).times` or something like that?  Not sure if Cypress has this.  Maybe also expect the inputAttendance button to be rendered?

Reply:
yes!